solr/core/src/test/org/apache/solr/util/SolrPluginUtilsTest.java

```diff
@@ -248,7 +248,7 @@ public void testDisjunctionMaxQueryParser() throws Exception {
         
   }
 
-  private static int countItems(@SuppressWarnings({"rawtypes"})Iterator i) {
+  private static int countItems(Iterator<?> i) {
```

Comment:
```suggestion
  private static int countItems(Iterator<Query> i) {
```

might work here, i haven't tried, but with quantity vs. quality considerations in mind in this cleanup and since this is a private static method in test code either is fine.

Reply:
I intentionally went with the more general approach here, because I wasn't sure if it would be useful in other place or not. While doing these massive refactorings, I have seen a lot of duplicate or near duplicate code so I'm starting to get a little more sensitive to it.